packages/lib/uuid.ts

```diff
@@ -18,4 +20,8 @@ export default {
 	createNanoForInboxEmail: (): string => {
 		return customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 8)();
 	},
+	uuidgen: (length = 22) => {
+		const uuidgenCustomAlphabet = createUuidgenCustomAlphabet(length);
+		return uuidgenCustomAlphabet();
+	},
```

Comment:
Actually we should move away from having big objects that includes multiple functions, and instead export the individual functions. So here it means leaving it as it was in uuidgen.ts, just have `export const uuiden () => { .... }` outside of this default object.

The reason is to potentially make use of tree shaking some day and generally to make the code more modular.